### PR TITLE
[Notifier] add notification assertion in notifier doc

### PR DESCRIPTION
**Pull request by MrYamous**
While reading testing documentation I came across some assertions for notifications. I think it can be useful to talk about this directly in Notifier documentation too, WDYT ?

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -759,6 +759,15 @@ typical alert levels, which you can implement immediately using:
...
         };
 
+Testing Notifier
+----------------
+
+With FrameworkBundle Symfony provide a :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait`
+which provide useful methods for testing your Notifier implementation.
+You can benefit from this class by using it directly or extending :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`.
+
+See :ref:`testing documentation <notifier-assertions>` for list of availables assertions.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
See :ref:`testing documentation <notifier-assertions>` for the list of available assertions.
```

File: notifier.rst [hunk 2]
```diff
@@ -759,6 +759,15 @@ typical alert levels, which you can implement immediately using:
             ;
         };
 
+Testing Notifier
+----------------
+
+With FrameworkBundle Symfony provide a :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait`
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
Symfony provides a :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait`
```

File: notifier.rst [hunk 3]
```diff
@@ -759,6 +759,15 @@ typical alert levels, which you can implement immediately using:
...
+which provide useful methods for testing your Notifier implementation.
+You can benefit from this class by using it directly or extending :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
You can benefit from this class by using it directly or extending the :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`.
```

File: notifier.rst [hunk 4]
```diff
@@ -759,6 +759,19 @@ typical alert levels, which you can implement immediately using:
...
+
+Symfony provides a :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait`
+which provide useful methods for testing your Notifier implementation.
+You can benefit from this class by using it directly or extending the :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`.
+
+See :ref:`testing documentation <notifier-assertions>` for the list of available assertions.
+
+.. versionadded:: 6.2
+
+    The Notifier assertions were introduced in Symfony 6.2.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait` was introduced in Symfony 6.2.
```

**Comment by MrYamous** [comment]
Thanks Oskar Stark for review 😃

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @MrYamous.

**Pull request merged**